test/test-main.js

```diff
@@ -0,0 +1,12 @@
+var main = require("./main");
+
+exports["test main"] = function(assert) {
+  assert.pass("Unit test running!");
```

Comment:
@irdan Nope, not yet. This week I have my finals, so I won't be able to look at it, I'll get back to you as soon as possible once I am done with my finals.